templates/posts/post_detail.html

```diff
@@ -187,6 +187,27 @@ <h1>{{ post.title }}</h1>
                   <div>
                     {{ comment.content|safe }}
                   </div>
+                  <form action="{% url 'reactions:like_comment' post.slug comment.id %}" method="post">
+                    {% csrf_token %}
+                    <button type="submit" style="border: none; background-color: transparent;">
+                      {% for comment_id, user_like in users_like.items %}
```

Comment:
in template better to have just render and all logic have inside view, there should be something like
```suggestion
    {% if comment.is_liked_by_user %}
      <i class="small material-icons" style="color: red;">favorite</i>
    {% else %}
       <i class="small material-icons" style="color: #fff;">favorite_border</i>
    {% endif %}
    {{ comment.like_count }}
    
```